dss_readspeaker.libraries.yml

```diff
@@ -1,3 +1,3 @@
 dss-readspeaker:
-  js:
```

Comment:
Do we need this file if we are dropping the js

Reply:
IIRC the libraries.yml is still needed in order to define a dependency to core/drupalSettings

Reply:
No, turns out it is not needed: File removed